bin/node/cli/src/chain_spec.rs

```diff
@@ -321,6 +322,12 @@ pub fn testnet_genesis(
 			max_members: 999,
 		}),
 		pallet_vesting: Some(Default::default()),
+		pallet_node_permission: Some(NodePermissionConfig {
+			nodes: vec![
+				ed25519::Public::from_str("5CibWAiYURpoy3o8SsA1hS75ypxCzGrT9mt3ufCerfQYCEC6").unwrap(),
+				ed25519::Public::from_str("5H1bTmUExYZZfHQ5S6RJ6k1qG4H7e1WHsimKXQc1Rz2j982o").unwrap(),
```

Comment:
Just curious, does this means by default only nodes with these two `ed25519`  public IDs could join? Are they dev node IDs?

Maybe more documentation on why these two IDs and how to customize them.

Reply:
It's in ed25519 ss58 format, I will try reuse existing keys, still I need to put the private key in doc so that user can run node with it.